apps/prairielearn/src/lib/db-types.ts

```diff
@@ -536,7 +546,13 @@ export const EnrollmentSchema = z.object({
   course_instance_id: IdSchema,
   created_at: DateFromISOString.nullable(),
   id: IdSchema,
-  user_id: IdSchema,
+  // View constraints on LTI and invitation states in 20250707220720_enrollments__meta_columns__add.sql
```

Comment:
I'm not a fan of referencing migrations because this will be out of date at some point when we make more changes. The constraints are always visible in `database/tables/enrollments.pg`, but we don't need to say that in a comment because it's obvious.